content/2024/sessions/__startofday.md

```diff
@@ -7,6 +7,5 @@ tags:
 videoId: null
 presentation: null
 draft: false
-type: sessions
```

Comment:
I don't quite understand why you have been removing this "type" line in some of the session of 2024? Did you do that intentionally, or did that happen by mistake?